diesel_cli/README.md

```diff
@@ -51,6 +51,10 @@ cannot connect to it. Finally it will create diesel's internal table for
 tracking which migrations have been run, and run any existing migrations if the
 internal table did not previously exist.
 
+#### `database reset`
+Drops the database specified in your DATABASE_URL if it can, and then runs
+`database setup`.
```

Comment:
The fact that this is in backticks makes me feel like this should be `diesel database setup`. Thoughts?